R/metric.R

```diff
@@ -26,28 +26,69 @@ histogram.jaccard <- function(x, y) {
 #' Kolmogorov-Smirnov divergence
 #' @param x histogram (vector of counts) 1
 #' @param y histogram (vector of counts) 2
-#' 
+#'
 #' @return the Kolmogorov-Smirnov divergence between x and y
 histogram.ks <- function(x, y) {
   max(abs(x - y), na.rm = TRUE)
 }
 
 #' Mean-squared error
-#' 
+#'
 #' @param x histogram (vector of counts) 1
 #' @param y histogram (vector of counts) 2
-#' 
+#'
 #' @return the mean-squared error of x and y
 histogram.mse <- function(x, y) {
   mean((x - y)^2)
 }
 
 #' Chi-squared
-#' 
+#'
 #' @param x histogram (vector of counts) 1
 #' @param y histogram (vector of counts) 2
-#' 
+#'
 #' @return the chi-squared distance between x and y
 histogram.chisq <- function(x, y) {
   sum((x - y)^2 / (x + y))
 }
+
+uniform.mle <- function(x, a, b, inclusive = TRUE, log = TRUE) {
+  UseMethod('uniform.mle')
+}
+
+# Internal, generalizes functionality between uniform.mle methods
+uniform.mle.helper <- function(x, x.start, x.end, a, b, inclusive = TRUE, log = TRUE) {
```

Comment:
Just out of curiosity, why did we use "uniform.mle" instead of "uniform_mle" for these functions?

Reply:
Eventually we will switch `_` to `.` for all function for BL standard.